ui/src/app/components/object-explorer/object-explorer.component.ts

```diff
@@ -174,6 +193,14 @@ export class ObjectExplorerComponent implements OnInit {
     this.sidenav.setDisplayRuleFlag(false)
   }
 
+  openAddSequenceForm(): void {
+    this.dialog.open(AddNewSequenceComponent, {
+      width: '30vw',
```

Comment:
Is it okay to hardcode these values? Will it work for all supported resolutions? Is there some 'base' component with some standard size that should be used instead?

Reply:
This is based on precedence where we have previously hardcoded such values. I can replace them with constants but as for if it will be supported by all resolutions, perhaps no, but we expect people to use this application in desktop mode so for that this is is fine